docker/Makefile

```diff
@@ -12,8 +12,8 @@ dev: start
 .PHONY: playwright-tests
 playwright-tests:
 
-    # stop any containers that are running
-	docker compose down
+    # stop any containers that are currently running
+	docker-compose down
```

Comment:
Done. Thanks. docker compose works, too. I mistakenly changed this line because the branch was behind the previous PR that added it. The Makefile currently contains a mix of both 'docker-compose' and 'docker compose' syntax.